help-view/bnd.bnd

```diff
@@ -0,0 +1 @@
+Vaadin-OSGi-Extender: true
```

Comment:
Such manifest template file generates `Bundle-SymbolicName: help-view` .
Which is not a serious issue but may be inconvenient since such name may be not unique.

Reply:
Probably might be an issue in larger projects. I'd go with defaults here and remove all optional configuration. Thus cleared everything the I assume is optional.